README.md

```diff
@@ -1,8 +1,15 @@
 # scikit-image: Image processing in Python
 
 [![Image.sc forum](https://img.shields.io/badge/dynamic/json.svg?label=forum&url=https%3A%2F%2Fforum.image.sc%2Ftags%2Fscikit-image.json&query=%24.topic_list.tags.0.topic_count&colorB=brightgreen&suffix=%20topics&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAOCAYAAAAfSC3RAAABPklEQVR42m3SyyqFURTA8Y2BER0TDyExZ+aSPIKUlPIITFzKeQWXwhBlQrmFgUzMMFLKZeguBu5y+//17dP3nc5vuPdee6299gohUYYaDGOyyACq4JmQVoFujOMR77hNfOAGM+hBOQqB9TjHD36xhAa04RCuuXeKOvwHVWIKL9jCK2bRiV284QgL8MwEjAneeo9VNOEaBhzALGtoRy02cIcWhE34jj5YxgW+E5Z4iTPkMYpPLCNY3hdOYEfNbKYdmNngZ1jyEzw7h7AIb3fRTQ95OAZ6yQpGYHMMtOTgouktYwxuXsHgWLLl+4x++Kx1FJrjLTagA77bTPvYgw1rRqY56e+w7GNYsqX6JfPwi7aR+Y5SA+BXtKIRfkfJAYgj14tpOF6+I46c4/cAM3UhM3JxyKsxiOIhH0IO6SH/A1Kb1WBeUjbkAAAAAElFTkSuQmCC)](https://forum.image.sc/tags/scikit-image)
-[![Stackoverflow](https://img.shields.io/badge/stackoverflow-Ask%20questions-blue.svg)](https://stackoverflow.com/questions/tagged/scikit-image)
-[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://skimage.zulipchat.com)
+[![Contributor forum](https://img.shields.io/badge/Scientific_Python-Contributor_forum-brightgreen?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAIGNIUk0AAHomAACAhAAA+gAAAIDoAAB1MAAA6mAAADqYAAAXcJy6UTwAAAHgUExURQAAAAD/AEC/QFygOFygOFOYNFygOFygOFyhOF2iLlygN1ygOFygOFyfOFmmM1ygOFmbN1ygOF2gO2WmT4C6mp3O5ZzM4YO7n53N5J3N5J3N5J3N5J3N5J3N5J3N5J3N5JXE1H2vsSNkOSRlO3+xtJ7L3p3N5H2wspm/vsHX026YfIqrkvP29Nzo5MHX1Jm/vX2ws////////////9Le1f///////xFUIZy3pP///6zDsh5dLW2Xd0p9Vh9eLhtbKglOGRpbKSJgMEyAWFicNlugOEGGLVKXNFOXNFygOFSYNEWKLjl/KS51JU+TMlWZNVyfOFCVM0qPMDd9KB5lHkKHLUWKLzuBKlGVM1CUMzh+KT+EKzB1JU6TMjR6JzuAKjJ4Jzd8KClvIz2DK12hOnOwdFidNjF3JRphHBlgHBFZGCFoH1SZNZ3N5JbI0V2hO1meNxBXF1KWNFygOXGvcH23j5zM4JXIzozCuInAsIW8oytvNzh5UZTHzJbI0JLGyDZ2Wjt6YJrL4JvL4dLf1uHp4+ju6e/z8Pz9/P////P29LzPwKS+qpKwmX+jiGmTc0B2TRdZJzZvRGmUc8rZzsXVyABIEQRLFfv8/Nnk3J25pDlxRmCNa0h8VCllNwZMFvbtL40AAABEdFJOUwAAAUDN97eKWQMusP4jBacG08vBWBnx8hiNjNnY+vnx8/L29vHy8MPw8ez59efx8MNmBtLRI+XzLBrQ0StbtfP+87FZrUzRuAAAAAFiS0dEMdnbHXIAAAAHdElNRQfoCQsVDSxxSaUKAAAA+ElEQVQY0y2PVVdCURQGPxMbu7tbwe5Gt9iiYne77e7WY+ex5a+6Lt55mTWPAwCwtbN3cNQ4QcXZxbW6xlBb56Zx97C2luqNxobGpmYiLQBPr5bWtnYydXR2mc3eAGx8unt6+/oHBoeGR0bHfAE///EJw+TU9MzsHBEFBCKI5xeIFpdMtLyyurYejBDmjc2t7Z3dvf0DZg5FGFs5PPp3OCKY+TgyKjomNu6EmeORkJiUnHJ6dn5xKVLT0jMyoRPi6vrm9u7+4fHp+UXooM8SKq9SvmXrgZxcpd4/Pr+kzMtXZgoKv39+LVJKWVSs/paUlpVbKiqrlPgDvJVMjOyXg/EAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjQtMDktMTFUMjE6MTM6MzErMDA6MDCZCUdBAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDI0LTA5LTExVDIxOjEzOjMxKzAwOjAw6FT//QAAACh0RVh0ZGF0ZTp0aW1lc3RhbXAAMjAyNC0wOS0xMVQyMToxMzo0NCswMDowMOe8+JwAAAAASUVORK5CYII=)](https://discuss.scientific-python.org/c/contributor/skimage)
```

Comment:
Uff, that's a big noisy wall of text at the beginning of the README if looking at it when it's not rendered. Is there a way to not encode the image directly?